src/endpoints/NotionEndpoint.php

```diff
@@ -2,28 +2,79 @@
 
 namespace viget\phonehome\endpoints;
 
+use DateTimeImmutable;
+use DateTimeZone;
+use Exception;
+use Notion\Databases\Database;
+use Notion\Databases\Properties\Date as DateDb;
+use Notion\Databases\Properties\MultiSelect as MultiSelectDb;
+use Notion\Databases\Properties\PropertyInterface;
+use Notion\Databases\Properties\RichTextProperty as RichTextDb;
+use Notion\Databases\Properties\Select as SelectDb;
+use Notion\Databases\Properties\Url as UrlDb;
 use Notion\Databases\Query;
 use Notion\Notion;
 use Notion\Pages\Page;
 use Notion\Pages\PageParent;
 use Notion\Pages\Properties\Date;
-use Notion\Pages\Properties\RichTextProperty;
+use Notion\Pages\Properties\MultiSelect;
+use Notion\Pages\Properties\Select;
 use Notion\Pages\Properties\Title;
 use Notion\Pages\Properties\Url;
-use viget\phonehome\models\SettingsNotion;
 use viget\phonehome\models\SitePayload;
+use viget\phonehome\models\SitePayloadPlugin;
 
 class NotionEndpoint implements EndpointInterface
 {
     private const PROPERTY_URL = "Url";
     private const PROPERTY_ENVIRONMENT = "Environment";
+    private const PROPERTY_CRAFT_EDITION = "Craft Edition";
     private const PROPERTY_CRAFT_VERSION = "Craft Version";
     private const PROPERTY_PHP_VERSION = "PHP Version";
     private const PROPERTY_DB_VERSION = "DB Version";
     private const PROPERTY_PLUGINS = "Plugins";
+    private const PROPERTY_PLUGIN_VERSIONS = "Plugin Versions";
     private const PROPERTY_MODULES = "Modules";
     private const PROPERTY_DATE_UPDATED = "Date Updated";
-    private const PROPERTY_TITLE = "Title";
+    private const PROPERTY_NAME = "Name";
+
+    /**
+     * @var array<string,array{
+     *   class: class-string<PropertyInterface>
+     * }>
+     */
+    private const PROPERTY_CONFIG = [
+        self::PROPERTY_URL => [
+            'class' => UrlDb::class,
+        ],
+        self::PROPERTY_ENVIRONMENT => [
+            'class' => SelectDb::class,
+        ],
+        self::PROPERTY_CRAFT_EDITION => [
+            'class' => SelectDb::class,
+        ],
+        self::PROPERTY_CRAFT_VERSION => [
+            'class' => SelectDb::class,
+        ],
+        self::PROPERTY_PHP_VERSION => [
+            'class' => SelectDb::class,
+        ],
+        self::PROPERTY_DB_VERSION => [
+            'class' => SelectDb::class,
+        ],
+        self::PROPERTY_PLUGINS => [
+            'class' => MultiSelectDb::class,
+        ],
+        self::PROPERTY_PLUGIN_VERSIONS => [
+            'class' => MultiSelectDb::class,
+        ],
```

Comment:
How sensitive is this to manipulations to the table? It might be possible to use a formula in a manually-created column but would that confuse the plugin?

Reply:
Custom columns seem to work fine. As long as their names don't conflict with what's in this list.